module/module.go

```diff
@@ -628,29 +607,17 @@ func (m *Module) RemoveResource(ctx context.Context, req *pb.RemoveResourceReque
 		return nil, err
 	}
 
-	// remove vidostreams when the camera resource implementing a passthrough is removed.
-	vcss, streamSourceByNameOk := m.streamSourceByName[name]
-	prs, activeResourceStreamsOk := m.activeResourceStreams[name]
-	if streamSourceByNameOk && activeResourceStreamsOk {
-		m.logger.CInfow(ctx, "calling RemoveTrack on", "name", name)
-		if err := m.pc.RemoveTrack(prs.sender); err != nil {
-			m.logger.CWarnw(ctx, "RemoveStream > RemoveTrack", "name", name, "err", err)
-		}
-		m.logger.CInfow(ctx, "calling Unsubscribe on", "name", name)
-		if err := vcss.Unsubscribe(ctx, prs.subID); err != nil {
-			m.logger.CWarnw(ctx, "RemoveStream > Unsubscribe", "name", name, "subID", prs.subID.String(), "err", err)
-		}
-		m.logger.CInfow(ctx, "deleting from activeResourceStreams", "name", name)
-		delete(m.activeResourceStreams, name)
-	}
-
 	if err := res.Close(ctx); err != nil {
 		m.logger.Error(err)
 	}
+	if err := coll.Remove(name); err != nil {
+		return nil, err
```

Comment:
Can we justify why it's correct for this case to not delete from `streamSourceByName` and `activeResourceStreams`?

Reply:
No I can not. moved.